src/services/settings/configService.js

```diff
@@ -0,0 +1,53 @@
+const { Configuration } = require('@schul-cloud/commons');
+const { authenticate } = require('@feathersjs/authentication');
+const reqlib = require('app-root-path').require;
+
+const { Forbidden } = reqlib('src/errors');
+const { hasRole } = require('../../hooks');
+
+const superheroAccess = async (context) => {
+	const isSuperHero = await hasRole(context, context.params.account.userId, 'superhero');
+	if (!isSuperHero) {
+		return Promise.reject(new Forbidden('You has no access.'));
+	}
+	return context;
+};
+
+const configServiceHooks = {
+	before: {
+		all: [authenticate('jwt'), authenticate('api-key'), superheroAccess],
```

Comment:
are you sure this works?
would like to see a test or something for this, especially since its the first time the api-key is used along with a different authentiaction method...

Reply:
Oh shit this should not committed as part of this PR, to often changed in the last time. -.-